jscomp/bin/bsppx.ml

```diff
@@ -2364,7 +2371,7 @@ let report_error ppf err =
 ;;
 
 let error_of_printer loc print x =
-  errorf ~loc "%a@?" print x
+  errorf_prefixed ~loc "%a@?" print x
```

Comment:
hi, you need update the OCaml submodule, we cherry-picked a commit from upstream